File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/IRemoteController.java
package com.example.try_gameengine.remotecontroller;

import android.graphics.Canvas;
import android.graphics.Paint;
import android.view.MotionEvent;

/**
 * @author irons
 * 
 */

public interface IRemoteController {
	public boolean onTouchEvent(MotionEvent event);

	public void drawRemoteController(Canvas canvas, Paint paint);
}

File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/IRemoteController.kt
package com.example.try_gameengine.remotecontroller

import android.graphics.Canvas
import android.graphics.Paint
import android.view.MotionEvent

/**
 * @author irons
 // */
interface IRemoteController {
    fun onTouchEvent(event: MotionEvent?): Boolean

    fun drawRemoteController(canvas: Canvas?, paint: Paint?)
}
